README.md

```diff
@@ -1,8 +1,34 @@
 # Community Air Quality Dashboard
+This project is a collaboration between the Harvey Mudd College Clinic Program and the South Gate Community Environmental Health Action Team (CEHAT). This repository has all the code for the front and back end of a community air quality monitoring dashboard, accessible at https://sg-cehat-air-quality.web.app. All sensor data is gathered using affordable [PurpleAir](https://www2.purpleair.com) sensors owned by the CEHAT. This website is tailored for the South Gate community, but can easily be used as a template for other communities with the appropriate modifications. One important feature of the website is its availability in both English and Spanish, which is relevant to the community of South Gate, but the langauges can easily be changed.
```

Comment:
```suggestion
# Community Air Quality Dashboard
This project is a collaboration between the Harvey Mudd College Clinic Program and the South Gate Community Environmental Health Action Team (CEHAT). This repository has all the code for the front and back end of a community air quality monitoring dashboard, accessible at [https://sg-cehat-air-quality.web.app](https://sg-cehat-air-quality.web.app). All sensor data is gathered using affordable [PurpleAir](https://www2.purpleair.com) sensors owned by the CEHAT. This website is tailored for the South Gate community, but it can be used as a template for other communities with the appropriate modifications. For more information about creating this website for another community, see `CLONING.md`. One important feature of the website is its availability in both English and Spanish, which is relevant to the community of South Gate, but other languages can be added with some code changes and by translating the website content into the other desired languages.
```